tuf/client/client.go

```diff
@@ -422,7 +420,8 @@ func (c *Client) downloadTargets(role string) error {
 }
 
 func (c *Client) downloadSigned(role string, size int64, expectedSha256 []byte) ([]byte, *data.Signed, error) {
-	raw, err := c.remote.GetMeta(role, size)
+	rolePath := utils.URLFilePath(role, expectedSha256)
```

Comment:
Is `URLFilePath` the name we want for this method? We're essentially assigning a "URL" to a path, and then sticking it into a getMeta.

Feels to me that the remote store should abstract that this is a remote call, so we shouldn't call this `URLFilePath`